src/main/java/com/google/impactdashboard/database_manager/data_read/DataReadManagerImpl.java

```diff
@@ -67,6 +86,26 @@ public double getAverageIAMBindingsInPastYear(String projectId) {
     }
   }
 
+  /**
+   * Returns the average number of bindings summed across every project belonging to
+   * the organization with id {@code organizationId} over however many days of data
+   * are in the IAM Bindings table. 
+   */
+  public double getOrganizationAvgBindingsInPastYear(String organizationId) {
+    QueryJobConfiguration queryConfiguration = queryConfigurationBuilder
+      .getAverageOrganizationBindingsConfiguration()
+      .addNamedParameter("organizationId", QueryParameterValue.string(organizationId))
+      .build();
+    TableResult results = database.readDatabase(queryConfiguration);
+    FieldValueList row = Iterables.getOnlyElement(results.iterateAll(), null);
+
+    if (row == null || row.get("AverageBindings").isNull()) {
```

Comment:
nit: simple if clause like this can be replaced by "return () ? 0.0 : row.get().getDoubleValue()";